docs/docusaurus.config.ts

```diff
@@ -100,18 +75,17 @@ const config: Config = {
             hideOnScroll: true,
             items: [
                 {
-                    type: 'docSidebar',
-                    sidebarId: 'tutorialSidebar',
+                    to: '/docs/getting-started/intro',
                     position: 'left',
                     label: 'Docs',
+                    activeBaseRegex: `/docs/`,
                 },
-                /*
                 {
-                    to: '/blog',
-                    label: 'Blog',
+                    to: '/api',
                     position: 'left',
+                    label: 'API Reference',
+                    activeBaseRegex: `/api/`,
                 },
-                */
                 {
                     href: 'https://discord.gg/GFzWFAAZcm',
                     position: 'right',
```

Comment:
somehow the discord logo has got messed up with this change

not sure how but maybe due to css styles or dependency change?

new:

![image](https://github.com/user-attachments/assets/c6bdf2de-22be-4940-a21a-8fc2d569403e)


before:

![image](https://github.com/user-attachments/assets/d943fee4-9c2d-44ed-b3bd-c5847fe734ec)